apiary.apib

```diff
@@ -5738,3 +5738,61 @@ From now on you won't be able to sign in to the **Direct Access** for specific p
                     "outBytesUncompressed": 0
                 }
             }
+
+# Group Development Branches
+
+**EXPERIMENTAL**
+
+Development branches allow you to prepare changes to configuration and tables in isolation. When ready, you can replace the production with the development branch.
+
+## Branches [/v2/storage/dev-branches/]
```

Comment:
Udělal bych to stejně jako to je u ostatních api callů. Ten vrácený job bych vrátil podle reality. Tj waiting + nevyplněné některé údaje.

Reply:
viz výše